src/content/pages/explore.mdx

```diff
@@ -0,0 +1,143 @@
+
+### For nature lovers
+
+- Petřín hill: Eiffel tower (in Prague), nicce view of the city
+- Vyšehrad: nice fortress next to the venue, park, you can sit and chill
+- Letná: viewpoint
+- Riegrovy Sady: park with nice view, there’s a rose garden there that most people don’t know about - you should check it out
+- Divoká šárka: small forest a bit further away, you can swim in Džbán (artificial lake)
+- Kunratický les: forest
+- Hostivář: area for walk, you can swim there as well
+
+*Use mapy.cz - there are routes there (red, blue and green ones) - the best maps for hiking in general*
+
+## Where to eat?
+
+Prague is a food city! There are many places you will find nice food, from traditional Czechia food to Vegan friendly food.
+
+### General recommendation
+
+Some yays and nays from our local volunteers:
+
+- 👎 Trdelnik is a tourist thing: nothing original or authentic
+- 👎 Pražská šunka on the street: tourist trap
+- 👍 Try Nakládaný hermelín (pickled camembert)
+- 👍 Smažený sýr (fried cheese)
+- 👍 Svíčková (meat with vegetable sauce and dumplings)
+- 👍 Guláš (goulash)
+
+
+### For veggie/ vegans
+
+We know some of us are veggie/ vegan, don't worrie you will find places that serve you veggie/ vegan food:
+
+- Shromazdiste: typical czech food in vegan versions, great food and great beer
+- Eateria: vegan junk food, best burger
+- Pastva: open during day only, european and healthy food
+- Dhaba beas: cheap buffet, indian and european food, it’s everywhere
+- Share Sweet & Espresso Bar: coffee shop with best vegan cakes
+
+According to HappyCow there are 89 vegan and vegetarian places to eat, so it’s not an issue. Typical Czech restaurants offer mostly only fried cheese and french fries, sometimes pickled camembert, but czech cuisine is mostly meat-heavy. If hungry during the night, go to any kebab place and you can order falafel wrap with veggies. You can also go to Burrito Loco - it’s mexican food chain, they offer vegan mana burrito and burrito with grilled veggies.
+
+[Visit happy cow vegan-vegetarian listings](https://www.happycow.net/europe/czech_republic/prague/?filters=vegan-vegetarian)
+
+## Where to drink?
+
+In European counties drinking is a popular way to socialise. Here are some recommendations:
+
+- For craft beer: Sibeeria bar, Pivotéka, BeerGeek, Zlý časy
+- Pilsner type beer: everywhere
+- For mild and not bitter Pilsner-style beer try Únětice (it’s in a few places), they also have 8 degrees with less alcohol
+- For more bitter try Plzeň/Pilsner or Kozel, Radegast
+For craft try Matuška (APA, IPA, ALE), Zíchovec (Sour ALE, APA, IPA, ALE) or Sibeeria (Sour ALE, APA, IPA, ALE)
+- For wine: Bublinka 2024 wine festival
+
+On top of the above:
+
+### All day social places
+
+- Automat Matuška
+
+### Recommendation for music lovers
+
+There are festivals around you can check them:
```

Comment:
```suggestion
There are multiple festivals going on you might want to check out:
```

Reply:
Do we have links to all the following festivals?

Has anybody checked if they are on between July 7th and 14th?